doc/source/contributing.rst

```diff
@@ -9,9 +9,12 @@ MDAnalysis is a free and open source project. It evolves and grows with the dema
 The development team very much welcomes contributions from its users. 
 Contributions can take many forms, such as:
 
-    * **bug reports** or **enhancement requests** filed through the `Issue Tracker`_
-    * **source code** for fixing bugs or new functionality (e.g. new analysis tasks)
-    * **documentation** (either in the code or on the user guide)
+    * **bug reports** or **enhancement requests** filed through the `Issue Tracker <https://github.com/MDAnalysis/mdanalysis/issues>`_
```

Comment:
Lots of moving things around and forgetting what was defined where 😪

Reply:
Unless you meant why inline links in general, which is because I change wording a lot and having to fix up the associated link doubles the work involved.